meshinfo/views/map.py

```diff
@@ -14,6 +14,14 @@
 from ..models import Link, Node
 from ..types import Band, LinkId, LinkStatus, LinkType, NodeStatus
 
+LINK_COLORS = {
+    "good": "#006164",
+    "ok": "#57c4ad",
+    "weak": "#edc656",
+    "poor": "#eda247",
+    "bad": "#db4325",
```

Comment:
These colors should be loaded from the application configuration, so that administrators can customize/override (but use these as the defaults).

Reply:
I opened #100 to address this.